scripts/globals/mobskills/whispers_of_ire.lua

```diff
@@ -0,0 +1,40 @@
+-----------------------------------
+-- Whispers of Ire
+-- Randomly absorbs 2 to 5 attributes from target.
+-- Type: Magical
+-- Notes: Can also use the ability to target itself and remove all debuffs (whispers_of_ire_self.lua)
+-----------------------------------
+require("scripts/globals/mobskills")
+require("scripts/globals/settings")
+require("scripts/globals/status")
+require("scripts/globals/msg")
+-----------------------------------
+local mobskill_object = {}
+
+mobskill_object.onMobSkillCheck = function(target, mob, skill)
+    return 0
+end
+
+mobskill_object.onMobWeaponSkill = function(target, mob, skill)
+    local amount = math.random(2, 5)
+    local count = 0
+
+    for i = 1, amount do
+        -- str down - chr down
+        local effectType = math.random(136, 142)
+
+        skill:setMsg(xi.mobskills.mobDrainAttribute(mob, target, effectType, 10, 3, 60))
+
+        count = count + 1
```

Comment:
if the math.random() rolled the same skill every time, would count still be incremented as the mob repeatedly steals strength? resulting in the output stating the mob drained 2-5 times, but only took strength